contracts/l2/dai.sol

```diff
@@ -72,7 +71,7 @@ contract Dai {
     DOMAIN_SEPARATOR = keccak256(abi.encode(
       keccak256("EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)"),
       keccak256(bytes(name)),
-      keccak256(bytes(version)),
```

Comment:
I prefer the named constant here. We should also update this to version "2"

Reply:
The reason I inlined this is because I don't see it present as a public constant in most other tokens. I'm okay to stick with the public constant though if you guys feel it's a better fit. Also, I was unclear on what necessitates a version bump here.

Reply:
Reading EIP 712, it looks like you should do a version bump on next major release which I think it's up for debate whether this is that. I would call this a minor release at most due to the breaking change on `permit`, but other than that it's pretty much the same.

Reply:
If we're following semver, that's a backwards-incompatible change, hence it requires a major version bump.

Reply:
Fair point.

Reply:
I've re-added the constant and set it to version 2.